examples/todo-two-levels/package.json

```diff
@@ -3,25 +3,30 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "react": "^16.1.1",
-    "react-dom": "^16.1.1",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
     "react-redux": "^5.0.6",
     "react-router": "^4.2.0",
     "react-router-dom": "^4.2.2",
     "redux": "^3.7.2",
-    "resolve-bus-memory": "^0.1.0",
-    "resolve-redux": "^0.1.0",
-    "resolve-scripts": "^0.1.0",
-    "resolve-storage-lite": "^0.1.0"
+    "resolve-bus-memory": "^0.2.1",
+    "resolve-redux": "^0.2.1",
+    "resolve-scripts": "0.2.1",
+    "resolve-storage-lite": "^0.2.1"
   },
   "scripts": {
-    "start": "resolve-scripts dev",
```

Comment:
Very good proposition! But now **CI** perform `start` script invocation, so `Jenkinsfile` should be changed too: here https://github.com/reimagined/resolve/blob/v0.2.2/examples/todo/tests/functional/testcafe_runner.js#L27 and here https://github.com/reimagined/resolve/blob/v0.2.2/examples/todo-two-levels/tests/functional/testcafe_runner.js#L27

Reply:
done